recipes/otree/recipe.yaml

```diff
@@ -0,0 +1,50 @@
+context:
+  version: "0.3.1"
+
+package:
+  name: otree
+  version: ${{ version }}
+
+source:
+  url: https://github.com/fioncat/otree/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: f71064cac1c7fc1047a119bdd2b630eac67008546d13b51b1ace1ab9b0314e02
+
+build:
```

Comment:
This is perfectly fine, but note that in many rust packages on conda-forge, we use the additional settings:

```
export CARGO_PROFILE_RELEASE_STRIP=symbols
export CARGO_PROFILE_RELEASE_LTO=fat
```

https://github.com/conda-forge/bat-feedstock/blob/main/recipe/build.sh#L5-L6
https://github.com/conda-forge/bat-feedstock/blob/main/recipe/bld.bat#L3-L4

Reply:
Thanks, @synapticarbors. I usually set these variables, but I didn't see them in any of the v1 recipes I used as template. I was assuming those were automatically set, as [`rattler-build`'s documentation mention that some environment variables are set automatically](https://rattler.build/latest/build_script/#environment-variables). If that's not the case, I can add these

Reply:
I'm not sure what `rattler-build` sets. You can always use `build.sh` and `build.bat` (note not `bld.bat`) as I did in:
https://github.com/conda-forge/jujutsu-feedstock/tree/main/recipe

Reply:
Thank you! I edited the recipe to set these variables in the `script` section:

```yaml
  script:
    env:
      CARGO_PROFILE_RELEASE_STRIP: symbols
      CARGO_PROFILE_RELEASE_LTO: fat
    - if: unix
      then:
        - cargo install --no-track --locked --root ${PREFIX} --path .
        - cargo-bundle-licenses --format yaml --output ${SRC_DIR}/THIRDPARTY.yml
    - if: win
      then:
        - cargo install --no-track --locked --root %PREFIX% --path .
        - if errorlevel 1 exit 1
        - cargo-bundle-licenses --format yaml --output %SRC_DIR%/THIRDPARTY.yml
        - if errorlevel 1 exit 1
```